### PR TITLE
[Tool] Add instructions on how to run debugger on FE to the developer doc

### PR DESCRIPTION
**Pull request by jiacheng-celonis**
## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [x] Tool

## Which issues of this PR fixes ：

N/A

## Problem Summary(Required) ：

N/A

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] I have added user document for my new feature or new function

### CODE REVIEW

File: docs/developers/development-environment/IDEA.md [hunk 1]
```diff
@@ -139,3 +139,14 @@ export LOG_DIR=/Users/smith/Code/starrocks/fe/log
 To standardize the coding style, you should import the `fe/starrocks_intellij_style.xml` code style file in IDEA.
 
 ![image-20220701193938856](../../assets/IDEA-2.png)
+
+## Attach debugger
+
+
+If you started the FE with the `debug` option, you can then attach IDEA debugger to the FE process.
```
Comment (chaoyli) [inline docs/developers/development-environment/IDEA.md:146]: If you started the FE with the debug option, you can then attach the IDEA debugger to the FE process.
```
./start_fe.sh --debug
```

See https://www.jetbrains.com/help/idea/attaching-to-local-process.html#attach-to-local.

**Review by imay — DISMISSED** [review]
LGTM

**Comment by wanpengfei-git** [comment]
starrocks_be_unittest succeeded.

**Review by imay — APPROVED** [review]

File: docs/developers/development-environment/IDEA.md (hunk 1 above)
Comment (jiacheng-celonis) [reply docs/developers/development-environment/IDEA.md]: @chaoyli DONE

**Review by chaoyli — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Pull request merged**